docker/compose/kafka-schema-registry.yml

```diff
@@ -62,5 +62,7 @@ services:
 
   ksql-cli:
       image: confluentinc/cp-ksql-cli:5.1.0
+      depends_on:
+        - ksql-server
```

Comment:
Does this add any value, it doesn't automatically brings up the KSQL-CLI anyway.